[FLINK-16057] Don't end input in ContinuousFileReaderOperator Benchmark

rkhachatryan opened the pull request:
Currently, END_OF_INPUT is sent once all splits were emitted.
Thus, all subsequent reads in ContinuousFileReaderOperator are made in
CLOSING state in a simple loop (MailboxExecutor.isIdle is always true).
Which is not a scenario this benchmark is intended for.

This change makes source to wait until all elements reach sink.

rkhachatryan committed:
[FLINK-16057] Don't end input in ContinuousFileReaderOperator Benchmark

Currently, END_OF_INPUT is sent once all splits were emitted.
Thus, all subsequent reads in ContinuousFileReaderOperator are made in
CLOSING state in a simple loop (MailboxExecutor.isIdle is always true).
Which is not a scenario this benchmark is intended for.

This change makes source to wait until all elements reach sink.

pnowojski commented on src/main/java/org/apache/flink/benchmark/ContinuousFileReaderOperatorBenchmark.java:
```diff
@@ -45,6 +51,7 @@
 
     private static final TimestampedFileInputSplit SPLIT = new TimestampedFileInputSplit(0, 0, new Path("."), 0, 0, new String[]{});
     private static final String LINE = Strings.repeat('0', 10);
+    private static OneShotLatch TARGET_COUNT_REACHED_LATCH = new OneShotLatch();
```
Could you copy the explanation from the commit message into a comment here, describing the intention behind this latch?

pnowojski commented:
Could you take a look at the build failure?

rkhachatryan committed:
[hotfix] Apply workaround for JDK.11 TLS issue

Prevent "No PSK available" error while fetching
artifacts from maven repo on travis.
See https://bugs.openjdk.java.net/browse/JDK-8213202

rkhachatryan commented:
Thanks for the feedback.
I've fixed the failure on travis and addressed the comment, merging.
